share/picongpu/examples/KelvinHelmholtz/etc/picongpu/16.cfg

```diff
@@ -58,18 +58,22 @@ TBG_ipngYX="--i_png.period 10 --i_png.axis yx --i_png.slicePoint 0.5 --i_png.fol
 TBG_eBin="--e_energyHistogram.period 100 --e_energyHistogram.filter all --e_energyHistogram.binCount 1024 --e_energyHistogram.minEnergy 0 --e_energyHistogram.maxEnergy 5000"
 TBG_iBin="--i_energyHistogram.period 100 --i_energyHistogram.filter all --i_energyHistogram.binCount 1024 --i_energyHistogram.minEnergy 0 --i_energyHistogram.maxEnergy 2000000"
 
-TBG_plugins="!TBG_ipngYZ                   \
-              !TBG_ipngYX                   \
-              !TBG_eBin                     \
-              !TBG_iBin                     \
-              !TBG_pngYX                    \
-              !TBG_pngYZ                    \
-              !TBG_hdf5                     \
-              --i_macroParticlesCount.period 100         \
-              --e_macroParticlesCount.period 100         \
-              --fields_energy.period 10     \
-              --e_energy.period 10 --e_energy.filter all \
-              --i_energy.period 10 --i_energy.filter all"
+TBG_openPMD="--openPMD.period 1000 \
+             --openPMD.file simData \
+             --openPMD.ext bp"
+
+TBG_plugins="!TBG_openPMD                  \
+             !TBG_ipngYX                   \
+             !TBG_eBin                     \
+             !TBG_iBin                     \
+             !TBG_pngYX                    \
+             !TBG_pngYZ                    \
+             !TBG_hdf5                     \
```

Comment:
please remove HDF5 if we use openPMD. There is no need to double the output data.